packages/components/src/internal/components/settings/NameIdSettings.tsx

```diff
@@ -47,18 +47,19 @@ interface NameIdSettingsProps extends InjectedRouteLeaveProps {
 interface State {
     allowUserSpecifiedNames: boolean;
     api?: ComponentsAPIWrapper;
+    confirm: string[];
     confirmCounterModalOpen?: boolean;
     confirmModalOpen: boolean;
-    error: string;
+    error: string[];
```

Comment:
Did you mean to change this? I don't see how/where `error` handling is changed.

Reply:
Oop nope, those keystrokes were meant for the new state